internal/pipeline/executor.go

```diff
@@ -308,8 +308,20 @@ func hydrate(p *pkg, hctx *hydrationContext) (resources []*yaml.RNode, err error
 func filterMetaData(resources []*yaml.RNode) []*yaml.RNode {
 	var filtered []*yaml.RNode
 	for _, r := range resources {
-		meta, _ := r.GetMeta()
-		if !strings.Contains(meta.APIVersion, "kpt.dev") {
+		m, _ := r.GetMeta()
+		if !strings.Contains(m.APIVersion, "kpt.dev") {
+			filtered = append(filtered, r)
+		}
+	}
+	return filtered
+}
+
+// filterFnConfig filters out function configs
+func filterFnConfig(resources []*yaml.RNode) []*yaml.RNode {
+	var filtered []*yaml.RNode
+	for _, r := range resources {
+		m, _ := r.GetMeta()
+		if _, ok := m.Annotations["config.kubernetes.io/function"]; !ok || m.Kind != "ConfigMap" {
```

Comment:
Somehow my brain took little bit more time to parse the above conditional logic :) consider writing it in selection form (seems simpler to read):

```Go
_, hasFnAnnotation := m.Annotations["config..."]
if m.Kind == "ConfigMap" && hasFnAnnotation {
 continue
}
filtered = append(...)
```

Reply:
done